src/modules/profile/profile.controller.ts

```diff
@@ -0,0 +1,102 @@
+import { APIResponse } from "#/src/lib/types/misc";
+import { toResponse } from "#/src/lib/utils";
+import { statusConst } from "#/src/lib/utils/status";
+import { Request as ExpressRequest } from "express"; // Explicitly import from Express
+import {
+  Controller,
+  Delete,
+  FormField,
+  Get,
+  Patch,
+  Request,
+  Route,
+  Security,
+  Tags,
+  UploadedFile,
+} from "tsoa";
+import { getReqUser } from "../auth/auth.helpers";
+
+import fileService from "../file/file.service";
+import { sanitizeUser } from "../user/user.helpers";
+import userService from "../user/user.service";
+import { SanitizedUser } from "../user/user.types";
+import { profileService } from "./profile.services";
```

Comment:
Always add a one line gap after imports
```suggestion
import { profileService } from "./profile.services";

```